### DISCUSSION TITLE
minimal install version

### DISCUSSION THREAD

**Discussion by JapCLoN** [cockpit-project/cockpit#16490] [Q&A] [answered]
Is it possible to se a minimal version just to report info back to main cockpit server, like CPU and RAM usage?

I have a few raspberry pi's which I have cockpit on, however on the many 1core version, cockpit takes up all CPU there is.

And often when trying to install cockpit on this small once, then it often ends on broken packages, mainly for pcp & cockpit-pcp.

Is there anyway to install bare minimum ?

**Comment by martinpitt** [comment, marked as answer]
The minimal package that you need is "cockpit-system" (which pulls in cockpit-bridge). With that, you can connect to this server from some remote cockpit installation. If you want the cockpit web server ("login page") directly on the Pi, install cockpit-ws as well.

That will avoid pcp and all the other goodies.

**Reply by JapCLoN** [reply]
Thank you very much!
